scripts/bundled_installer

```diff
@@ -50,7 +51,8 @@ function install_ebcli() {
     if [ -f ebcli_installer.py ]; then
         ${PYENV_BIN}/python ./ebcli_installer.py \
             --python-installation ${PYENV_BIN}/python \
-            --virtualenv-executable ${PYENV_BIN}/virtualenv
+            --virtualenv-executable ${PYENV_BIN}/virtualenv \
+            --script-source ${SCRIPT_SOURCE}
```

Comment:
The responsibility of printing the Python `PATH`-`export` message must not be that of `ebcli_installer.py`. It should be that of the `bundled_installer`.

The way we should deal with printing the Python PATH-export message is as follows:

1. pass a flag to `python_installer`, `--suppress-path-export-message`, so that it can avoid displaying the message
2. print the export message that `python_installer` in `bundled_installer` if and only if Python was actually installed (`python_installer` might have found Python already in PATH and hence skipped installation). For this, you can set a boolean in python_installer.
3. you should ensure that even if `ebcli_installer` fails, the Python export statement is printed. I am concerned here that the error message in red followed by the green + yellow Python PATH-export messages might make the customer believe that everything went alright, which I think, is why I was hesitant to put the PATH-export messages for both Python and the EBCLI together in the first place.

Reply:
> 2.  ... If there is a way to do this that I'm unaware of then this approach also makes sense.

You can pass arguments to `source python_installer`.

> 3. ... However I think that concern would support the idea of displaying neither export statement.

The concern supports the idea of not making the change at all so that the python_installer continue printing the PATH-export message where it currently is. Note, however, I am not insisting we retain the behaviour.

> 3. ... What usecase would require a user to want to add python to path without also having succeeded the ebcli installer?

That's a fair point. Considering the python_installer is exclusive to the ebcli_installation, we can avoid showing the Python PATH-export statement even if Python was installed.

Reply:
Just updated the commit to reflect these changes